pysatNASA/instruments/maven_insitu_kp.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Full license can be found in License.md
+# Full author list can be found in .zenodo.json file
+# DOI:10.5281/zenodo.3986131
+#
+# DISTRIBUTION STATEMENT A: Approved for public release. Distribution is
+# unlimited.
+# ----------------------------------------------------------------------------
+"""Module for the MAVEN insitu instruments.
+
+Supports the in situ Key Parameter (kp) data from multiple instruments
+onboard the Mars Atmosphere and Volatile Evolution (MAVEN) satellite.
+
+Accesses local data in CDF format.
+Downloads from CDAWeb.
+
+Properties
+----------
+platform
+    'maven'
+name
+    'insitu_kp'
+tag
+    None supported
+inst_id
+    None supported
+
+
+Examples
+--------
+::
+    import pysat
+
+    insitu = pysat.Instrument(platform='maven', name='insitu_kp')
+    insitu.download(dt.datetime(2020, 1, 1), dt.datetime(2020, 1, 31))
+    insitu.load(2020, 1, use_header=True)
+
+"""
+
+import datetime as dt
+import functools
+
+from pysat.instruments.methods import general as mm_gen
+from pysatNASA.instruments.methods import cdaweb as cdw
+from pysatNASA.instruments.methods import general as mm_nasa
+from pysatNASA.instruments.methods import maven as mm_mvn
+
+# ----------------------------------------------------------------------------
+# Instrument attributes
+
+platform = 'maven'
+name = 'insitu_kp'
+tags = {'': 'in situ Key Parameter data'}
```

Comment:
mixed case

Reply:
I believe "Key Parameter" is a proper name, and so allowed to be capitalized.  @jklenzing @t-esman please correct me if I'm wrong.